x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/compare_fields_table/get_incompatible_mappings_table_columns/index.tsx

```diff
@@ -28,11 +28,7 @@ export const getIncompatibleMappingsTableColumns = (): Array<
   {
     field: 'type',
     name: i18n.ECS_MAPPING_TYPE_EXPECTED,
-    render: (type: string) => (
-      <CodeSuccess data-test-subj="codeSuccess">
-        {type != null ? type : EMPTY_PLACEHOLDER}
```

Comment:
we remove this check because it's an impossible state after updated types. `getIncompatibleMappingsTableColumns` only accepts `incompatible` fields which  by definition always have `type` field.